mmlapipe/pose/multi_pose_demo.json

```diff
@@ -0,0 +1,190 @@
+      },
+      "package": "mmlapipe"
+    },
+    {
+      "registry_name": "YOLOVideo",
+      "name": "vid2",
+      "kwargs": {
+        "video_src": "https://cdn.videvo.net/videvo_files/video/free/2023-02/large_watermarked/221026_02_Dancer_4k_042_preview.mp4",
+        "width": 640,
+        "height": 480,
+        "frame_key": "frame",
+        "frame_rate": 30,
+        "include_meta": true,
+        "loop": true
+      },
+      "package": "mmlapipe"
+    },
+    {
+      "registry_name": "YOLOVideo",
+      "name": "vid3",
+      "kwargs": {
+        "video_src": "https://joy1.videvo.net/videvo_files/video/free/video0482/large_watermarked/_import_62bbefe51f3547.77164811_preview.mp4",
+        "width": 640,
+        "height": 480,
+        "frame_key": "frame",
+        "frame_rate": 30,
+        "include_meta": true,
+        "loop": true
+      },
+      "package": "mmlapipe"
+    },
+    {
+      "registry_name": "MultiPoseNode",
+      "name": "pose",
+      "kwargs": {
+        "task": "-pose",
+        "scale": "n",
+        "device": "cuda"
+      },
+      "package": "mmlapipe"
+    },
+    {
+      "registry_name": "MultiSaveNode",
+      "name": "save",
+      "kwargs": {
+        "source_key": "vid",
+        "filename": "testing",
+        "file_format": "df"
+      },
+      "package": "mmlapipe"
+    },
+    {
+      "registry_name": "MultiSaveNode",
+      "name": "save1",
+      "kwargs": {
+        "source_key": "vid1",
+        "filename": "testing",
+        "file_format": "vid"
+      },
+      "package": "mmlapipe"
+    },
+    {
+      "registry_name": "MultiSaveNode",
+      "name": "save3",
+      "kwargs": {
+        "source_key": "vid3",
+        "filename": "testing",
+        "file_format": "df"
+      },
+      "package": "mmlapipe"
+    },
+    {
+      "registry_name": "MultiSaveNode",
+      "name": "save2",
+      "kwargs": {
+        "source_key": "vid2",
+        "filename": "testing",
+        "file_format": "vid"
+      },
+      "package": "mmlapipe"
+    },
+    {
+      "registry_name": "DisplayNode",
+      "name": "display",
+      "kwargs": {
+        "frames_key": "frame"
+      },
+      "package": "mmlapipe"
+    }
+  ],
+  "adj": [
+    [
+      "vid",
+      "pose"
+    ],
+    [
+      "vid1",
+      "pose"
+    ],
+    [
+      "vid2",
+      "pose"
+    ],
+    [
+      "vid3",
+      "pose"
+    ],
+    [
+      "pose",
+      "save"
+    ],
+    [
+      "pose",
+      "save1"
+    ],
+    [
+      "pose",
+      "save2"
+    ],
+    [
+      "pose",
+      "save3"
+    ],
+    [
+      "pose",
+      "display"
+    ]
+  ],
+  "manager_config": {
+    "logdir": "cp-logs",
+    "port": 9001
+  },
+  "mappings": {
+    "local": [
+      "vid1",
+      "pose",
+      "save",
+      "save1",
+      "save2",
+      "save3",
+      "vid",
+      "vid2",
+      "vid3",
+      "display"
+    ]
+  },
+  "timeouts": {
+    "commit_timeout": 120,
+    "shutdown_timeout": 300
+  }
+}
```

Comment:
No new line at the end of file. Additionally, this needs to be moved in `configs/` directory at the root of the repo